scripts/create-data.js

```diff
@@ -1165,20 +1179,20 @@ const createUserAndColonyData = async () => {
     );
 
     // verify users
-    await graphqlRequest(
-      updateColonyMetadata,
-      {
-        input: {
-          id: newColonyAddress,
-          isWhitelistActivated: true,
-          whitelistedAddresses: Object.keys(availableUsers.randomUsers).slice(
-            0,
-            noOfMembers,
-          ),
-        },
-      },
-      GRAPHQL_URI,
-      API_KEY,
+    await Promise.all(
+      Object.keys(availableUsers.walletUsers).map(async (userAddress) => {
```

Comment:
Just a heads up that I think `availableUsers` doesn't include the first dev wallet (leela), which means that the colony creator is not actually verified

![Screenshot from 2024-01-30 14-35-06](https://github.com/JoinColony/colonyCDapp/assets/1193222/316462a5-1bcf-4aa2-a6a2-0a02e80c7789)

This however only affects the creation script, and not the actual user / colony creation wizard in the CDapp

But it's nice to have it here as well :)